tests/e2e/tests/plugins/PhpPlugin.spec.ts

```diff
@@ -91,6 +91,12 @@ suite(`The 'PhpPlugin' tests`, async () => {
         test('Run debug', async () => {
             await topMenu.selectOption('View', 'Debug');
             await ide.waitLeftToolbarButton(LeftToolbarButton.Debug);
+
+            // workaround for the issue: https://github.com/eclipse/che/issues/20067
```

Comment:
https://github.com/eclipse/che/issues/20067 has already resolved.
I assume it's workaround to https://github.com/eclipse/che/issues/20046

Reply:
Fixed, thx.